ops_boba/cloudformation/gnosis-transaction-service-mainnet.yaml

```diff
@@ -0,0 +1,400 @@
+              HardLimit: 65536
+            - Name: core
+              SoftLimit: 0
+              HardLimit: 0
+          EnvironmentFiles:
+            - Value: !Sub
+              - "arn:aws:s3:::${S3BucketV2}/gnosis-safe-mainnet.env"
+              - S3BucketV2:
+                  Fn::ImportValue: !Sub "${InfrastructureStackName}:S3BucketV2"
+              Type: "s3"
+          Environment:
+            - Name: ETH_L2_NETWORK
+              Value: 1
+            - Name: PYTHONPATH
+              Value: /app/
+            - Name: DJANGO_SETTINGS_MODULE
+              Value: config.settings.production
+            - Name: C_FORCE_ROOT
+              Value: true
+            - Name: DJANGO_ALLOWED_HOSTS
+              Value: "*,.localhost,127.0.0.1,[::1],safe-client-gateway,safe-client.rinkeby.boba.network"
+            - Name: ETHEREUM_NODE_URL
+              Value: "https://lightning-replica.boba.network/"
+            - Name: ETHEREUM_TRACING_NODE_URL
+              Value: "https://lightning-replica.boba.network/"
+            - Name: REDIS_URL
+              Value: !Sub
+                - "redis://${EndpointAddress}:6379/0"
+                - EndpointAddress:
+                    Fn::ImportValue: !Sub "${InfrastructureStackName}:RedisClusterGnosis:DNS"
+            - Name: CELERY_BROKER_URL
+              Value: !Sub
+                - "redis://${EndpointAddress}:6379/1"
+                - EndpointAddress:
+                    Fn::ImportValue: !Sub "${InfrastructureStackName}:RedisClusterGnosis:DNS"
+          Command:
+            - "./docker/web/celery/scheduler/run.sh"
+          DockerLabels:
+            com.datadoghq.ad.check_names: '["worker"]'
+            com.datadoghq.ad.init_configs: '[{}]'
+            com.datadoghq.ad.logs: '[{"source": "worker", "service": "worker"}]'
+          Essential: true
+          Memory: "4096"
+      NetworkMode: awsvpc
+      TaskRoleArn: !GetAtt TaskRole.Arn
+      ExecutionRoleArn: "arn:aws:iam::942431445534:role/ecs-s3-rolw"
+      Family: !Sub "Gnosis-${EnvironmentName}"
+  ECSService:
+    Type: AWS::ECS::Service
+    Properties:
+      Cluster:
+        Fn::ImportValue: !Sub "${InfrastructureStackName}:EcsClusterGnosis"
+      LoadBalancers:
+        - ContainerName: nginx-safe-transaction-service
+          ContainerPort: 8000
+          TargetGroupArn: !Ref GnosisLBTargetGroup
+      NetworkConfiguration:
+        AwsvpcConfiguration:
+          AssignPublicIp: DISABLED
+          SecurityGroups:
+            - Fn::ImportValue: !Sub "${InfrastructureStackName}:LoadBalancerGnosis:SecurityGroup"
+          Subnets:
+            - Fn::ImportValue: !Sub "${InfrastructureStackName}:PrivateSubnet1"
+            - Fn::ImportValue: !Sub "${InfrastructureStackName}:PrivateSubnet2"
+      HealthCheckGracePeriodSeconds: 300
+      DesiredCount: !Ref ServiceCount
+      LaunchType: EC2
+      SchedulingStrategy: REPLICA
+      ServiceName: "gnosis-safe-transaction-service"
```

Comment:
Consider not naming your ECS services, as updating them when there is a replacement required is not possible if they are named.

Reply:
If you leave out ServiceName it will derive a randomly generated name from the name you have given the resource at it's top level

Reply:
Oh yes, I've seen this happening!